infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -38,6 +38,10 @@ private StatusLogger(final String name) {
 
   public void onStartup(final String version) {
     log.info("Teku version: {}", version);
+    log.info(
```

Comment:
just wonder if we should bump this to a warning...

Reply:
I don't think there's a better example of an informational message. :) Warn implies something the user should be concerned about but this is just routine information they may need to know.

Reply:
who pays attention to info messages.... :)